system-xml.md

```diff
@@ -0,0 +1,472 @@
+# system.xml reference
+
+- [Overview](#overview)
+- [Tabs](#tabs)
+    - [Tab attribute reference](#tab-attribute-reference)
+    - [Tab node reference](#tab-node-reference)
+    - [Example: Create a new tab](#example-create-a-new-tab)
+- [Sections](#sections)
+    - [Section attribute reference](#section-attribute-reference)
+    - [Section node reference](#section-node-reference)
+    - [Example: Create a new section](#example-create-a-new-section)
+- [Groups](#groups)
+    - [Group attribute reference](#group-attribute-reference)
+    - [Group node reference](#group-node-reference)
+    - [Example: Create a new group](#example-create-a-new-group)
+- [Fields](#fields)
+    - [Field attribute reference](#field-attribute-reference)
+    - [Field node reference](#field-node-reference)
+    - [Example: Create a new field](#example-create-a-new-field)
+    - [Common Source Models](#common-source-models)
+    - [Field Validation](#field-validation)
+
+## Overview
+
+The `system.xml` file allows you to manage the Magento system configuration. Use this topic as a general reference for
```

Comment:
This should explain that it is package-related, not global.

Reply:
I think we need to clarify here that system.xml is associated to a Magento package. Additionally, the configuration inheritance strategy should at least also be mentioned here. system.xml's merge together, but it's not obvious from this doc that this happens. At the very least, a link to such a doc would be nice.